spec/factories/loans.rb

```diff
@@ -55,11 +55,12 @@
 
 
     trait :active do
-      status_option_id Loan::STATUS_OPTIONS.value_for('Active')
+      # note: can't seem to count on the seed.db data having been loaded when these are executed
+      status_option_id 1 #Loan.status_option_set.value_for_label('Active')
```

Comment:
I agree with moving the majority of what's currently in `seeds.rb` out somewhere else (I think that's what I recommended way back when it started being populated), but I'm not sure about seeding the options/option sets, I'm still trying to think through an alternative way to model this, we have a lot of different constraints on these that aren't being met well by the current schema. 

Sometimes they're just arbitrary options for populating a dropdown, sometimes they're filterable, sometimes you need to be able to filter by multiple at once, sometimes you need to filter by default, sometimes you only want a subset of the option set to be filterable by the public, but you want the full range filterable by the admin users, sometimes they're potentially security-sensitive (like public/hidden). It might be that we're dealing with two or more *different* kinds of things and we just need to split them up between which options are arbitrary and can be stored in a generic table and which have too much business logic to be meaningful in a vacuum.

Seeding the root division is okay, but I actually think we can get by without the rest of it. Even for the root division, once we start building out the specs and figure out what actual domain functionality is tied to that conceptually, we'll probably have a better idea of how to best deal with that in test suites. It might involve making sure it gets seeded from the `seeds.rb`, but it's possible that won't be necessary and that we'll want to do something more specific with that. In the meantime, all the specs that rely on a division are generating one via association, so until there's a need to test specifically for the *root* division (this will probably come when we're generating policy specs for authorization, and related functionality), we're probably fine to leave it out of the specs.